src/mpsc/blocking.rs

```diff
@@ -410,6 +410,129 @@ feature! {
         pub fn try_send(&self, val: T) -> Result<(), TrySendError<T>> {
             self.core.try_send(self.slots, val, self.recycle)
         }
+        /// Returns the *total* capacity of the channel for this [`Sender`].
```

Comment:
let's add a line of whitespace here:
```suggestion
        }

        /// Returns the *total* capacity of the channel for this [`Sender`].
```